tox.ini

```diff
@@ -7,43 +7,37 @@ testdeps =
 
 [tox]
 envlist =
-    lint
-    py27
-    py36
-    coverage
+    {py27,py36}
+    py36-lint
+    py36-coverage
+    py36-docs
```

Comment:
Does the 'docs' env name matters for readthedoc ?

Reply:
readthedocs doesn't use tox, so it shouldn't be a problem